packages/custom_lint/test/src/workspace_test.dart

```diff
@@ -714,19 +714,22 @@ void main() {
         );
       }
 
-      test('throws MissingPubspecError if package does not contain a pubspec',
-          () async {
-        final workspace = await createSimpleWorkspace([]);
-        workspace.dir('package').createSync(recursive: true);
+      test(
```

Comment:
We'll need a new test for the changes to avoid regreessions

Reply:
I've edited this one to test for the issue that is fixed.